util/upload-cfn-templates.py

```diff
@@ -23,6 +25,19 @@ def get_all_aws_regions(region):
     return set(sorted(r.get("RegionName") for r in ec2.describe_regions().get("Regions")))
 
 
+def get_template_extension(templates_dir, template_name):
+    matching_files = glob("{}{}.*".format(templates_dir, template_name))
+    if len(matching_files) != 1:
```

Comment:
Is this to exclude that, by chance, both the .json and the .yaml template exist for the same template_name?
minor: I am wondering, why doesn't this script simply use a map to recognize the only valid template files expected for substacks?